src/plugins/template/src/sync_infer_request.hpp

```diff
@@ -39,6 +39,7 @@ class InferRequest : public ov::ISyncInferRequest {
     void start_pipeline();
     void wait_pipeline();
     void infer_postprocess();
+    void terminate();
```

Comment:
why is it not called `cancel`? I suppose users should be able to cancel sync requests as well

Reply:
It is not a part of public API.